baseplate/integration/pyramid.py

```diff
@@ -86,34 +86,88 @@ class ServerSpanInitialized(BaseplateEvent):
     pass
 
 
+class HeaderTrustHandler(object):
+    """Abstract class used by :py:class:`BaseplateConfigurator` to validate headers.
+    See :py:class:`StaticTrustHandler` for the default implementation.
+    """
+
+    def should_trust_trace_headers(self, request):
+        """ Whether baseplate should parse the trace headers from the inbound request
+
+        :param request pyramid.util.Request: The request
+
+        :returns bool: Whether baseplate should parse the trace headers from the inbound request.
+        """
+        raise NotImplementedError
+
+    def should_trust_edge_context_payload(self, request):
+        """ Whether baseplate should trust the edge context headers from the inbound request.
```

Comment:
This should probably say `edge context payload` or `edge context header` rather than `headers`.